packages/pigeon/test/pigeon_lib_test.dart

```diff
@@ -349,8 +349,8 @@ void main() {
 
   test('nested enum', () {
     final Pigeon dartle = Pigeon.setup();
-    final ParseResults parseResult =
-        dartle.parseFile(thisPath, types: <Type>[NestedEnumApi]);
+    final ParseResults parseResult = dartle.parseFile(thisPath,
+        types: <Type>[NestedEnumApi], ignoresInvalidImports: true);
```

Comment:
Why are almost all of the tests running in a way that's different from how things will actually be run?

Reply:
Historical reasons.  I don't want to change the tests and the code at the same time because it's easier to validate that we aren't breaking anything this way.